src/main/java/recipeditor/parser/Parser.java

```diff
@@ -229,26 +251,22 @@ private static Command parseFindCommand(String[] parsed) {
     }
 
 
-    public static Command parseHelpCommand(String[] parsed) {
-
-        if (parsed.length == COMMAND_INPUT_LENGTH) {
-            logger.log(Level.INFO, "Help command initialised");
-            return new HelpCommand(parsed[1]);
-        }
-        return new InvalidCommand(HelpCommand.CORRECT_FORMAT + HelpCommand.HELP_MESSAGE);
+    public static Command parseHelpCommand(String params) {
+        logger.log(Level.INFO, "Help command initialised");
+        return new HelpCommand(params);
```

Comment:
the original change account for the case of excess arguments in `HelpCommand`, so you do not want to report issue when `HelpCommand` contains excess arguments?